dll/ntdll/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ list(APPEND SOURCE
     ldr/ldrpe.c
     ldr/ldrutils.c
     ldr/verifier.c
+    rtl/hashtable.c
```

Comment:
All i386 and amd64 builds are failing, as expected.

Reply:
Not sure what to do for this.

Reply:
Maybe something similar to
https://github.com/reactos/reactos/blob/9e96571a268625242128db7e9769602eb907e257/dll/win32/uxtheme/CMakeLists.txt#L22

Note:
With/Without this, your next TODOs remain valid: fix the C code so compiling it (as 7+) would succeed.